tests/integration/WireProtocolTest.cpp

```diff
@@ -25,9 +25,7 @@ class MockCukeEngine : public CukeEngine {
     EXPECT_THAT(typeid(classname).name(), StrEq(typeid(expression).name()))
```

Comment:
I think `EXPECT_TYPE` was only needed for `EXPECT_PTRTYPE`. Guess we can remove it now.